src/cocoa/toga_cocoa/window.py

```diff
@@ -18,9 +18,9 @@ def toolbar_identifier(cmd):
 
 
 class CocoaViewport:
-    def __init__(self, view):
+    def __init__(self, view, dpi):
         self.view = view
-        self.dpi = DISPLAY_DPI
```

Comment:
If DISPLAY_DPI is a full constant, then there's no real need to pass it around as an argument. The argument only serves to make more copies of something.